src/components/PurchaseOrder/PO.js

```diff
@@ -155,21 +158,44 @@ class PO extends Component {
     </Button>
   );
 
+  mountModal = () => {
+    this.setState({ isCloseModalOpen: true });
+  }
+
+  unmountModal = () => {
+    this.setState({ isCloseModalOpen: false });
+  }
+
   render() {
     const { location, history, match, mutator, resources, parentResources } = this.props;
     const order = get(resources, ['order', 'records', 0]);
     const orderId = get(order, 'id');
     const orderNumber = get(order, 'po_number', '');
     const poLines = get(order, 'po_lines', []);
+    const workflowStatus = get(order, 'workflow_status', '');
```

Comment:
what is the purpose of default value as empty string here? At the same time you don't do this below at View component https://github.com/folio-org/ui-orders/pull/162/files#diff-279a209f4fc176681655f2d27f9265cfR18